features/promotion/applying_catalog_promotions/reapplying_catalog_promotions_after_adding_its_action.feature

```diff
@@ -0,0 +1,22 @@
+@applying_catalog_promotions
+Feature: Reapplying catalog promotion after editing its action
```

Comment:
In the second feature file you also have scenarios with adding actions, I would be for merging these two files into one